Enigmatry.Entry.SmartEnums.EntityFramework/ModelConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,12 @@
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore;
+using SmartEnum.EFCore;
+
+namespace Enigmatry.Entry.SmartEnums.EntityFramework;
+
+[PublicAPI]
+public static class ModelConfigurationBuilderExtensions
+{
+    public static void EntryConfigureSmartEnum(this ModelConfigurationBuilder configurationBuilder)
+        => configurationBuilder.ConfigureSmartEnum();
```

Comment:
According to the Ardalis.EfCore documentation. Through method EntryConfigureSmartEnum we can add more configuration conventions as needed.